deploy/cache-writer.yaml

```diff
@@ -18,7 +18,166 @@ kind: Template
 metadata:
   name: ccx-cache-writer
 objects:
-# TODO: Add the cache-writer deployment + services
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: valkey-writer
+    annotations:
+      ignore-check.kube-linter.io/minimum-three-replicas: "This app doesn't have that much traffic"
+  spec:
+    inMemoryDb: true
+    envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ccx
+    deployments:
+      - name: db-writer
```

Comment:
Do we want to reuse this name?

Reply:
Tbh, I don't know where is this name actually used. Pod name is different.

Reply:
it's the deployment name. If you make a `oc get deployments` you will see it's called `ccx-cache-writer-db-writer`, which comes from `{CLOWDAPP_NAME}-{DEPLOYMENT_NAME}`. In this case it will be `valkey-writer-db-writer`.

Reply:
You can even reuse the clowdapp for cache-writer having a `redis-writer` and another `valkey-writer` deployments

Reply:
I mean, instead of having 2 clowdapps in the Template, having 2 deployments in the clowdapp

Reply:
Ah, I see. But I would probably leave it as it is.